web/src/helpers/theme.ts

```diff
@@ -32,6 +32,12 @@ export const theme = createTheme({
     error: {
       main: '#ee7b7b'
     },
+    warning: {
+      main: '#7D7D7D'
+    },
```

Comment:
Typically, this would be an orange or yellow in a design system.

Reply:
I was referencing the colors defined in [`helpers/runs.ts`](https://github.com/MarquezProject/marquez/blob/main/web/src/helpers/runs.ts). Also, maybe I should just use `runColorMap` defined in `runs.ts`?

Reply:
Yeah, let's go with that.